tenant_schemas/middleware.py

```diff
@@ -88,10 +88,9 @@ class MyTenantMiddleware(DefaultTenantMiddleware):
     DEFAULT_SCHEMA_NAME = None
 
     def get_tenant(self, model, hostname, request):
-        if self.DEFAULT_SCHEMA_NAME is None:
+        schema_name = self.DEFAULT_SCHEMA_NAME
```

Comment:
Could you move this into the except? since it doesn't need to be done before every tenant is selected. only when it cannot be found.